tutorials/fiori-ios-scpms-starter-mission-03/fiori-ios-scpms-starter-mission-03.md

```diff
@@ -336,7 +383,7 @@ The SAP Fiori for iOS Search Bar control inherits is using the standard `UISearc
 
 3. The code won't compile at the moment because you haven't conformed to the `UISearchResultsUpdating` protocol. We will fix that at a later point but first call the `setupSearchBar()` method right below the `loadData()` call in the `viewDidLoad()` method.
```

Comment:
Please don't say "we", say "you" instead as described in the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqgno1sd).